src/Operators/pointwisestencil.jl

```diff
@@ -0,0 +1,425 @@
+#= 
```

Comment:
Can we add some docs here briefly describing the strategy/process?

Reply:
Definitely. I didn't make documentation or tests because Simon wants this available for others asap, but I'll definitely add those soon.